docs/index.html

```diff
@@ -357,6 +367,68 @@ <h5>Browser extensions</h5>
         </dd>
     </dl>
 
+    <h5>SSH Agent</h5>
+    <dl>
+        <dt id="faq-ssh-agent-how"><a href="#faq-ssh-agent-how">How does the SSH Agent work?</a></dt>
+        <dd>
+            The SSH Agent feature is supported on all target platforms (Linux, macOS and Windows) and it acts as a client
+            for an existing agent despite its name. It can automatically add SSH keys from your KeePassXC database when
```

Comment:
"add SSH keys from your KeePassXC database *to a running SSH agent*"

and then remove the part about "despite its name"